packages/next/server/config-shared.ts

```diff
@@ -1,13 +1,15 @@
 import os from 'os'
 import { Header, Redirect, Rewrite } from '../lib/load-custom-routes'
-import { ImageConfig, imageConfigDefault } from './image-config'
+import {
+  ImageConfig,
+  ImageConfigComplete,
+  imageConfigDefault,
+} from './image-config'
 
-type NoOptionals<T> = {
-  [P in keyof T]-?: T[P]
```

Comment:
Equals to [`Required<Type>
`](https://www.typescriptlang.org/docs/handbook/utility-types.html).